app/controllers/api/v1/reference/reasons_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Reference
+      class ReasonsController < ApiController
```

Comment:
Without context, the word `reason` is not easy to understand.
It'd be nice name it `MoveReasons`